documentation/System_Network_Setup.md

```diff
@@ -13,6 +13,21 @@ to be set up, as subsequent adaptation can be very time-consuming.
 With the chosen ‘Layer3-Underlay-BGP-to-the-Host’, the eBGP protocol is used to efficiently (especially for larger environments)
 organise the the path-descision of packets between nodes.
 
+Unlike a classic layer 2 network, this provides the following advantages:
+
+- No more Multichassis-LACP, no Spanning-Tree complexities
+- A scalable setup for larger or critical environments with a high number of nodes
+  (layer 2 is simpler and more efficient; with a high number of nodes, layer 3 underlays cause a number of complex problems that layer 2 does not)
+- Connections can be transported and routed with fewer problems and dynamically over different network paths depending on saturation, 
+  preference and the availability of the same
+- BGP/Anycast allows the IP addresses of services to be made concurrently available at several locations.
+- Support for fast convergence of routing state across the network
+- Support for draining traffic from a node to be taken down
+- Support for filtering inbound and outbound advertisement
+- Tenant traffic is encapsulated in Geneve or VXLAN in the packets transported via the infrastructure 
+  (the configuration of the network switches can be kept very simple)
+- Future: Support for establishing EVPN (tenant) connections using the eBGP
+
```

Comment:
OpenStack docs contain a nice page where both L2 and L3 limitations and benefits are summarized: https://docs.openstack.org/arch-design/design-networking/design-networking-concepts.html

IMO the marked concern of L2 in the case of large L2 broadcast domain is constant churn in MAC tables and network traffic overhead caused by ARP. 

On the other hand, the limitation of L3 mentioned in the doc: "layer-3 networking include the need to design an IP addressing scheme rather than relying on the switches to keep track of the MAC addresses automatically" could be solved by BGP unnumbered feature (IIUTC).

Reply:
Thanks for the feedback and the reference to the document.
I updated the documentation at bit to clarify that a bit.
Especially the BGP IPv6 unnumbered is something what i missed :-)